projects/components/src/form-field/form-field.component.ts

```diff
@@ -65,6 +65,9 @@ export class FormFieldComponent {
   @Input()
   public showBorder: boolean = false;
 
+  @Input()
+  public showOnlyErrorBorder: boolean = false;
```

Comment:
why this change?

Reply:
To show only red borders

Reply:
To show only red borders

Reply:
We need consistent behavior across forms. Why do we want to show borders but not error message?

Reply:
Yes, I agree but exist some cases where it would be good to show the red borders without messages, for example when I want to show two nested fields with the same message.

Reply:
In that case, we should be combining two fields inside the same form-field component

Reply:
So should I use an object (with a pair of variables inside) instead of showing the input separatly?

Reply:
showOnlyErrorBorder  and showTransparentBackground were Removed